contracts/exchange/GRVTExchange.sol

```diff
@@ -22,7 +24,13 @@ contract GRVTExchange is
     _disableInitializers();
   }
 
-  function initialize(address admin, address chainSubmitter, address initializeConfigSigner) public initializer {
+  function initialize(
```

Comment:
would be good to add a comment explaining each of the variable passed in

Reply:
Added comment